admin/pkg/authtoken/authtoken.go

```diff
@@ -58,6 +58,20 @@ func NewRandom(t Type) *Token {
 	return &tkn
 }
 
+// NewToken generates a token from a UUID and secret.
+func NewToken(t Type, id uuid.UUID, secret [24]byte) *Token {
+	return &Token{
+		Type:   t,
+		ID:     id,
+		Secret: secret,
+	}
+}
+
+// TokenString returns the token as a string.
+func (t *Token) TokenString() string {
+	return fmt.Sprintf("%s_%s_%s", Prefix, t.Type, marshalBase62(append(t.ID[:], t.Secret[:]...)))
+}
```

Comment:
Seems like it's not used anywhere?

Reply:
Yeah just added and testing it out for ListTokens API, not needed now, removing